model_cards/gpt2-README.md

```diff
@@ -1,10 +1,147 @@
 ---
+language: english
 tags:
 - exbert
 
 license: mit
 ---
 
+
+# GPT-2
+
+Pretrained model on English language using a causal language modeling (CLM) objective. It was introduced in
+[this paper](https://d4mucfpksywv.cloudfront.net/better-language-models/language_models_are_unsupervised_multitask_learners.pdf)
+and first released at [this page](https://openai.com/blog/better-language-models/).
+
+Disclaimer: The team releasing GPT-2 did not write a model card for this model so this model card has been written by
+the Hugging Face team.
```

Comment:
They _did_ write a model card in their original repo: https://github.com/openai/gpt-2/blob/master/model_card.md (cc @thomwolf)

Would be nice to merge the two if some information is missing from ours.

Reply:
Oh I did not see that, will merge the two of them and adapt the disclaimer.